src/scores/categorical/contingency_impl.py

```diff
@@ -754,8 +754,41 @@ def heidke_skill_score(self) -> xr.DataArray:
         Returns:
             xr.DataArray: An xarray object containing the Heidke skill score
 
+
+        .. math::
+            \\\\
+            \\text{HSS} =
+                \\frac{\\text{true positives} + \\text{true negatives} - E_{random}}{
+                       \\text{total count} - E_{\\text{random}}}
+
+        where
+
+        .. math::
+            \\begin{aligned}
+                &E_{\\text{random}}
+                \\\\&= \\text{expected correct matches due to random chance}
+                \\\\&= \\frac{1}{\\text{total count}} \\cdot
+                        \\left[
+                            \\begin{array}{rc1}
+                            \\left(\\text{true positives} + \\text{false negatives}\\right)
+                            \\cdot
+                            \\left(\\text{true positives} + \\text{false positives}\\right)
+                            \\\\
+                            + \\left(\\text{true negatives} + \\text{false negatives}\\right)
+                            \\cdot
+                            \\left(\\text{true negatives} + \\text{false positives}\\right)
+                            \\end{array}
+                        \\right]
+            \\end{aligned}
+
+
         Notes:
             - Range: -1 to 1, 0 indicates no skill. Perfect score: 1.
+            - HSS = Heidke Skill Score
+            - "True positives" is the same as "hits".
+            - "False negatives" is the same as "misses".
+            - "False positives" is the same as "false alarms".
+            - "True negatives" is the same as "correct negatives".
```

Comment:
.. math::
            \\begin{aligned}
                &E_{\\text{random}}
                \\\\&= \\text{expected correct matches due to random chance}
                \\\\&= \\frac{1}{\\text{total count}} \\cdot
                        \\left[
                            \\begin{array}{rc1}
                            \\left(\\text{tp} + \\text{fn}\\right)
                            \\cdot
                            \\left(\\text{tp} + \\text{fp}\\right)
                            + \\left(\\text{tn} + \\text{fn}\\right)
                            \\cdot
                            \\left(\\text{tn} + \\text{fp}\\right)
                            \\end{array}
                        \\right]
            \\end{aligned}


        Notes:
            - Range: -1 to 1, 0 indicates no skill. Perfect score: 1.
            - HSS = Heidke Skill Score
            - "True positives" (:math:`\\text{tp}`) is the same as "hits".
            - "False negatives" (:math:`\\text{fn}`) is the same as "misses".
            - "False positives" (:math:`\\text{fp}`) is the same as "false alarms".
            - "True negatives" (:math:`\\text{tn}`) is the same as "correct negatives".

Reply:
I'm not sure, but I wonder if this is easier to read perhaps?

Reply:
I was thinking of that as an alternative. If you find that easier to read I will use that. In that case I can also omit the array alignment, which was mainly to accommodate the long equation.

Reply:
Thanks @tennlee I've updated the screenshot - see if you're happy with this version.

Reply:
Thanks, vey happy with this